docs/asciidoctorj.adoc

```diff
@@ -39,7 +40,7 @@ ifdef::env-github[]
 image::http://img.shields.io/travis/asciidoctor/asciidoctorj/master.svg[Build Status,90,18,link=https://travis-ci.org/asciidoctor/asciidoctorj]
 endif::[]
 
-ifdef::env-site,env-browser[]
+ifeval::["{toc}" == "macro"]
```

Comment:
Are you saying we need to change this line now or we should change this line (one day for "correctness")